entity/src/vulnerability.rs

```diff
@@ -1,7 +1,8 @@
 use crate::{advisory_vulnerability, vulnerability_description};
 use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel, Deserialize, Serialize)]
```

Comment:
If I understand correctly, then we're directly exposing the database structure here. I would really want to avoid that.

Even though we only have three fields: having id and identifier is already confusing. Exposing any internal database ID doesn't make much sense IMHO.

Reply:
Ah, good catch. I'll create a DTO.